spec/functional/provider/whyrun_safe_ruby_block_spec.rb

```diff
@@ -43,7 +43,7 @@
     end
 
     it "updates the evil laugh, even in why-run mode" do
-      new_resource.run_action(new_resource.action)
+      Array(new_resource.action).each {|action| new_resource.run_action(action) }
```

Comment:
Shouldn't `new_resource.action` already be returning an array here?  When would it not?